### PR TITLE
[Doc-Improvement][Update-channeldata-in-personal-conversation][2129904]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/bot-v3/bot-conversations/bots-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/521595763aca139ca51514bec32eb659ac696c12/msteams-platform/resources/bot-v3/bot-conversations/bots-conversations.md) | 83 | 46 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/41488d98-c2fc-488b-a924-0a247a62094c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/bot-v3/bot-conversations/bots-conversations.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ When interacting in a channel, your bot should be smart about taking certain con
 
 The `channelData` object contains Teams-specific information and is the definitive source for team and channel IDs. You should cache and use these ids as keys for local storage.
 
-The `channelData` object is not included in messages in personal conversations since these take place outside of any channel.
+The `channelData` object include only `tenant.id` in messages for personal conversations since these take place outside of any channel.
```
Comment (v-chetsh) [inline msteams-platform/resources/bot-v3/bot-conversations/bots-conversations.md:155]: Looks like this change is not correct.
Please wait, we will confirm once and let you know.

**Review by v-chetsh — CHANGES_REQUESTED** [review]
Reviewed and provided my comment.

**Comment by ashishguptaiitb** [comment]
Hi @v-habibaumme , please help me understand what is the next action required to complete this update?

**Comment by v-chetsh** [comment]
@v-npaladugu - Could you please remove the below line from documentation:
Doc section for ref : https://docs.microsoft.com/en-us/microsoftteams/platform/resources/bot-v3/bot-conversations/bots-conversations#teams-channel-data

UI
image

File: msteams-platform/resources/bot-v3/bot-conversations/bots-conversations.md [hunk 2]
```diff
@@ -152,7 +152,7 @@ When interacting in a channel, your bot should be smart about taking certain con
 
 The `channelData` object contains Teams-specific information and is the definitive source for team and channel IDs. You should cache and use these ids as keys for local storage.
 
-The `channelData` object is not included in messages in personal conversations since these take place outside of any channel.
```
Comment (v-chetsh) [inline msteams-platform/resources/bot-v3/bot-conversations/bots-conversations.md:155]: @v-npaladugu - Could you please remove the below line from documentation:
Doc section for ref : https://docs.microsoft.com/en-us/microsoftteams/platform/resources/bot-v3/bot-conversations/bots-conversations#teams-channel-data

UI
image

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/bot-v3/bot-conversations/bots-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a9a4d731b319293ceb6b214df55d0f5c7a334917/msteams-platform/resources/bot-v3/bot-conversations/bots-conversations.md) | 83 | 45 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/389830ca-bba6-41b2-8d6f-d55997fe3104) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

**Review by v-habibaumme — APPROVED** [review]
Approved

**Review by ashishguptaiitb — APPROVED** [review]

**Review by v-chetsh — APPROVED** [review]
Changes looks fine, approving it.

**Pull request merged**